example/src/Main.purs

```diff
@@ -72,7 +72,7 @@ ui = parentComponent render eval
                         )
                         Nothing
                 , initialState :
-                    { key : Nothing, editor : Nothing }
+                    initialAceState
```

Comment:
Using `aceConstructor` for the whole slot definition is another option here. Not sure if it improves things though, what do you think?